features/call/impl/src/main/kotlin/io/element/android/features/call/impl/services/CallForegroundService.kt

```diff
@@ -33,8 +35,12 @@ import timber.log.Timber
 class CallForegroundService : Service() {
     companion object {
         fun start(context: Context) {
-            val intent = Intent(context, CallForegroundService::class.java)
-            ContextCompat.startForegroundService(context, intent)
+            if (ContextCompat.checkSelfPermission(context, Manifest.permission.RECORD_AUDIO) == PackageManager.PERMISSION_GRANTED) {
```

Comment:
Really weird corner case, but what happens if the user didn't enable the microphone permission and joined the room anyway? Will the audio from the other side still be heard, even when the device is locked? I assume it would work for the app just being in background since the PiP window is alive.